src/System.Management.Automation/FormatAndOutput/common/PSStyle.cs

```diff
@@ -307,6 +307,11 @@ public sealed class FormattingData
             /// </summary>
             public string FormatAccent { get; set; } = "\x1b[32;1m";
 
+            /// <summary>
+            /// Gets or sets the style for table headers.
+            /// </summary>
+            public string TableHeader { get; set; } = "\x1b[32;1m";
```

Comment:
this might be a problem if the screen background is 32;1m, yes?

Reply:
This is the default, it's customizable for the user